db/migrate/20220623165020_create_projects.rb

```diff
@@ -0,0 +1,10 @@
+class CreateProjects < ActiveRecord::Migration[7.0]
+  def change
+    enable_extension "pgcrypto" unless extension_enabled?("pgcrypto")
+    
+    create_table :projects, id: :uuid do |t|
+      t.integer :urn # Every URN really is an integer, not a string which only contains numerical characters.
```

Comment:
As `urn` is so important to the model I wonder if we should:
- require it in the db and the applicaiton (validate the model)
- put an index on it